pipeline-runner/R/qc-3-filter_high_mito.R

```diff
@@ -19,45 +19,45 @@
 #' @export
 #' @return a list with the filtered seurat object by mitochondrial content, the config and the plot values
 #'
-filter_high_mito <- function(scdata, config, sample_id, cells_id, task_name = "mitochondrialContent", num_cells_to_downsample = 6000) {
-  cells_id.sample <- cells_id[[sample_id]]
+filter_high_mito <- function(scdata_list, config, sample_id, cells_id, task_name = "mitochondrialContent", num_cells_to_downsample = 6000) {
```

Comment:
similarly here if we don't overwrite the default values for `task_name` and `num_cells_to_downsample`, we better make them into constants.